src/main/java/com/openlattice/edm/PostgresEdmManager.java

```diff
@@ -59,7 +75,8 @@
  * @author Matthew Tamayo-Rios &lt;matthew@openlattice.com&gt;
  */
 public class PostgresEdmManager implements DbEdmManager {
-    private static final Logger logger = LoggerFactory.getLogger( PostgresEdmManager.class );
+    private static final String getAllEntitySets = "SELECT * FROM " + PostgresTable.ENTITY_SETS.getName();
```

Comment:
Not a blocking comment but do we plan on eventually having a single file of all of our PG queries somewhere? Might be nice to have a `ReadQueries.getAllEntitySets` vs `WriteQueries.updateEntityMetadata`

Reply:
Most of them are in PostgresQueries except for specific queries that are contained in *QueryService classes.